source/catalog.html

```diff
@@ -43,15 +42,15 @@
     </nav>
   </header>
   <main class="page-main">
-    <h1>Каталог продукции</h1>
+    <h1 class="page-main__title">Каталог продукции</h1>
     <section class="catalog">
       <h2 class="visually-hidden">Основные товары</h2>
       <div class="catalog__wrapper">
         <ul class="catalog__list">
           <li class="catalog__item">
             <article class="goods">
-              <a class="goods__link-title" href="#">
-                <h3>Cat Energy PRO 500 г</h3>
+              <a class="goods__link" href="#">
+                <h3 class="goods--title">Cat Energy PRO 500 г</h3>
```

Comment:
Кажется тут опечатка вышла, это должен быть элемент.
```suggestion
                <h3 class="goods__title">Cat Energy PRO 500 г</h3>
```
И то же самое во всех местах.